rotkehlchen/api/rest.py

```diff
@@ -1108,11 +1109,23 @@ def add_history_events(self, events: list['HistoryBaseEntry']) -> Response:
         return api_response(result_dict, status_code=HTTPStatus.OK)
 
     def edit_history_events(self, events: list['HistoryBaseEntry']) -> Response:
-        db = DBHistoryEvents(self.rotkehlchen.data.db)
-        try:
+        events_db = DBHistoryEvents(self.rotkehlchen.data.db)
+        if events[0].entry_type == HistoryBaseEntryType.ASSET_MOVEMENT_EVENT:
+            try:
```

Comment:
EDIT: Probably should just be removed as per what I wrote in the other part.

Two things.

1. Explain why you do this here.
2. Why not just move this to `edit_history_event`? To save conditionals?
```suggestion
            try:  # Explain here
```

Reply:
As discussed in discord moved this to `rest_helpers/history_events.py`